source/administration/changelog.md

```diff
@@ -2,9 +2,17 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released bi-monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
-## Release v3.6.0  
+## Release v3.6.1  
 
-Release date: 2017-01-16
+### Notes on Patch Release
+
+ - **v3.6.1, released 2017-01-17**
+   - Fixed a performance regression when sending many notifications at once (for example, when `@all` or `@channel` is used in a channel with many users)
+   - Fixed an issue where the config flag for the CLI was not backwards compatible
```

Comment:
There's different alternatives for the config flag, including `--config` (new) and `-config` (old). I think it's appropriate to just reference it as the "config flag", unless we want to reword the whole sentence.